plugins/extraction/extract_env.ml

```diff
@@ -410,12 +410,12 @@ and extract_module access venv env mp ~all mb =
   let impl = match Mod_declarations.mod_expr mb with
     | Abstract -> error_no_module_expr mp
     | Algebraic me -> extract_mexpression access venv env mp (mod_type mb) me
-    | Struct sign ->
+    | Struct (reso,sign) ->
```

Comment:
I think you should just ignore it there but leave some FIXME comment.

Reply:
Why?

Reply:
To allow future fix relying on the resolver but let the test-suite go through locally.

Reply:
This isn't what's breaking the test suite, it breaks in coqchk

Reply:
OK, so fix the other one above instead.